src/test/java/edu/wpi/grip/ui/controllers/MainWindowControllerTest.java

```diff
@@ -15,15 +17,18 @@
     @Override
     public void start(Stage stage) throws Exception {
         new Main().start(stage);
+
+        final PaletteView palette = lookup(".palette").<PaletteView>queryFirst();
+        palette.operationsProperty().setAll(new AdditionOperation());
```

Comment:
This is designed to test the entire UI in its full setup with all operations loaded.